backend/utils/file_storage/helpers/prompt_studio_file_helper.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any, Union
+
+from file_management.exceptions import OrgIdNotValid
+from utils.file_storage.common_utils import FileStorageUtil
+from utils.file_storage.constants import FileStorageConstants, FileStorageType
+from utils.file_storage.helpers.common_file_helper import FileStorageHelper
+
+
+class PromptStudioFileHelper:
+    @staticmethod
+    def handle_sub_directory_for_prompt_studio(
```

Comment:
NIT: I suggest renaming the function to make it less verbose and accurately describe what it does. 
1. Since the function is a part of `PromptStudioFileHelper` class I don't think we need to suffix with `_for_prompt_studio`
1. `handle_sub_directory_` is a bit vague and we can't guess what it does without checking the definition. I suggest something of the form `PromptStudioFileHelper.get_prompt_studio_dir(org_id, user_id, ps_id, is_create=True)`